peps/numerical.rst

```diff
@@ -0,0 +1,3295 @@
+.. _numerical-index:
+
+Numerical Index
```

Comment:
This file shouldn't exist in the repo, maybe we're missing a gitignore?

Reply:
Fixed